### PR TITLE
Add supported services

### PR DESCRIPTION
**Pull request by jnimmo**
To document the new alarm_keypress and alarm_output_control services in PR home-assistant/home-assistant#3932.

### CODE REVIEW

File: source/_components/envisalink.markdown [hunk 1]
```diff
@@ -69,3 +69,11 @@ Configuration variables:
 - **zones** (*Optional*): Envisalink boards have no way to tell us which zones are actually in use, so each zone must be configured in Home Assistant.  For each zone, at least a name must be given. *Note: if no zones are specified, Home Assistant will not load any binary_sensor components.*
 - **partitions** (*Optional*): Again, Envisalink boards do not tell us what is in use and what is not, so each partition must be configured with a partition name. If no partition parameter is specified, then no alarm_panel or sensor components are loaded.
 
+Supported services:
+
+- **alarm_disarm**
```
Comment (Landrash) [inline source/_components/envisalink.markdown]: This would need some descriptive text also. Looks very bare and like it's missing without.

**Comment by jnimmo** [comment]
Thanks @Landrash, hopefully this is a bit more descriptive?

**Comment by Landrash** [comment]
Looks good. Changed your description to point to the new pull request.
Can be merged when the related pull request is merged! :cookie:

**Comment by Landrash** [comment]
@jnimmo I'm assuming your merged pull request is correct with this documentation? If so I'll merge this one.

**Comment by jnimmo** [comment]
Thanks @Landrash - yes it is all correct now. Cheers

**Pull request merged**